src/main/resources/application.yml

```
@@ -0,0 +1,6 @@
+spring:
+  datasource:
+    url: jdbc:h2:mem:emotionalHelp
+    username: team3
+    password: qwerty
+    driverClassName: org.h2.Driver
```

```suggestion
    driverClassName: org.h2.Driver
  h2:
    console.enabled: true      
```
Let's add this property so we can access our H2 database from browser:
1. Open http://localhost:8080/h2-console
2. JDBC URL: jdbc:h2:mem:emotionalHelp
3. Username: team3
4. Password: qwerty

---

Thanks, it's gonna be very useful